config/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Ignore config files, but not the samples
+*.txt
```

Comment:
This ignores *all* txt files, right? Wouldn't that affect stuff like `strings/` folder?

Reply:
The gitignore is in the folder @pali6